.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,9 @@
 <!-- *Before creating an issue please make sure you are using the latest version of mongoose -->
 
+<!-- *Having an issue connecting to mongoose? Check here https://mongoosejs.com/docs/faq.html#operation-buffering-timed-out -->
```

Comment:
"Getting a `Operation `...` timed out after 10000 ms` error? Read this FAQ entry: ..."